src/perftest_resources.c

```diff
@@ -338,8 +311,14 @@ static inline int _new_post_send(struct pingpong_context *ctx,
 	default:
 		fprintf(stderr, "Post send failed: unknown operation code.\n");;
 	}
-
-	if (qpt == IBV_QPT_UD) {
+	if (qpt == IBV_QPT_DRIVER) {
```

Comment:
IBV_QPT_DRIVER doesn't necessarily imply DC, SRD for ex. uses it as well (the code already exists, maybe these patches aren't rebased).
This function should test for connection type instead.

Reply:
This will be covered by the backport of the https://github.com/linux-rdma/perftest/pull/74